src/react-queries/queries/swapper.ts

```diff
@@ -0,0 +1,33 @@
+import { createQueryKeys } from '@lukemorales/query-key-factory'
+import type { ChainId } from '@shapeshiftoss/caip'
+import { type HDWallet, supportsETH } from '@shapeshiftoss/hdwallet-core'
+import type { GetEvmTradeQuoteInput } from '@shapeshiftoss/swapper'
+import { arbitrumBridgeApi } from 'lib/swapper/swappers/ArbitrumBridgeSwapper/endpoints'
+import { getTradeQuote } from 'lib/swapper/swappers/ArbitrumBridgeSwapper/getTradeQuote/getTradeQuote'
+
+// Only used for arbitrum bridge swapper use outside of swapper for now
+export const swapper = createQueryKeys('swapper', {
+  arbitrumBridgeTradeQuote: (
```

Comment:
To make @woodenfurniture happy (and this cleaner, we don't need to bring a whole trade quote here, just to fetch the swap): let's consume `fetchArbitrumBridgeSwap` instead of `getTradeQuote`

Reply:
Nvm, gave it a try but this won't do, `getUnsignedEvmTransaction` needs a proper `tradeQuote`

Reply:
Appreciate the effort and intent ser 🙏

Reply:
Appreciate the intent and effort here ser 🙏